tests/expression_converter_/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
```

Comment:
Is there a reason this dir is named with a trailing underscore (`tests/expression_converter_/`)?

Reply:
That's the convention @timgraham came up with for [this](https://github.com/mongodb-forks/django/commit/a571a2ca49b834bfefcba3b2cd5eec1c4ea08b82). It may be the case that any test package name that doesn't conflict with a Django test package does not need the `_` but I haven't tested that and it may be more confusing to have some with and some without the `_`!